### PR TITLE
Update CDI JNDI tests and repeat against EE10

### PR DESCRIPTION
**Pull request by tevans78**
#build

### CODE REVIEW

File: dev/com.ibm.ws.cdi.jndi_fat/fat/src/com/ibm/ws/cdi/jndi/lookup/BeanManagerLookupServlet.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    @Inject
+    MyBean myRequestScopedBean;
+
+    private BeanManager getBeanMangerViaJNDI() throws Exception {
+        return (BeanManager) new InitialContext().lookup("java:comp/BeanManager");
+    }
+
+    public void testBeanManager(String beanManagerDescription, BeanManager beanManager) {
+        assertNotNull(beanManagerDescription + " was null", beanManager);
```
Comment (Azquelt) [inline dev/com.ibm.ws.cdi.jndi_fat/fat/src/com/ibm/ws/cdi/jndi/lookup/BeanManagerLookupServlet.java:45]: Missing `@Test` annotation

**Review by Azquelt — CHANGES_REQUESTED** [review]
Missing test annotation, otherwise I think this is good

File: dev/com.ibm.ws.cdi.jndi_fat/fat/src/com/ibm/ws/cdi/jndi/lookup/BeanManagerLookupServlet.java (hunk 1 above)
Comment (tevans78) [reply dev/com.ibm.ws.cdi.jndi_fat/fat/src/com/ibm/ws/cdi/jndi/lookup/BeanManagerLookupServlet.java:45]: doh!

Comment (tevans78) [reply dev/com.ibm.ws.cdi.jndi_fat/fat/src/com/ibm/ws/cdi/jndi/lookup/BeanManagerLookupServlet.java:45]: ah, no, that is a common test method, used by the other tests ... perhaps it should not be public?

Comment (Azquelt) [reply dev/com.ibm.ws.cdi.jndi_fat/fat/src/com/ibm/ws/cdi/jndi/lookup/BeanManagerLookupServlet.java:45]: Oh, so it is, yes make it non-public

**Review by Azquelt — APPROVED** [review]

**Pull request merged**